gobcore/model/gobmodel.json

```diff
@@ -5,103 +5,83 @@
     "attributes": {
       "meetboutid": {
         "type": "GOB.String",
-        "source_mapping": "BOUT_NR",
         "description": "Unieke identificatie van de meetbout"
       },
       "xcoordinaat": {
         "type": "GOB.Decimal",
-        "source_mapping": "BOUT_XCOORD",
         "decimal_separator": ",",
```

Comment:
Isn't decimal_separator part of the import mapping as well? We use '.' as decimal separator internally right?

Reply:
Ah, yes. Same goes for format, indeed.